addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -150,10 +150,10 @@ def _default_company_id(self):
     day_close = fields.Float(compute='_compute_day', string="Days to Close")
     delay_close = fields.Float(compute="_compute_day", string='Delay to Close', readonly=True, group_operator="avg", help="Number of days to close", store=True)
     color = fields.Integer("Color Index", default=0)
-    emp_id = fields.Many2one('hr.employee', string="Employee", tracking=True, help="Employee linked to the applicant.")
+    emp_id = fields.Many2one('hr.employee', string="Employee", help="Employee linked to the applicant.")
     user_email = fields.Char(related='user_id.email', type="char", string="User Email", readonly=True)
     attachment_number = fields.Integer(compute='_get_attachment_number', string="Number of Attachments")
-    employee_name = fields.Char(related='emp_id.name', string="Employee Name", readonly=False)
+    employee_name = fields.Char(related='emp_id.name', string="Employee Name", readonly=False, tracking=False)
```

Comment:
tracking=False should be implicit.

Reply:
I had to explicitly untrack it, it was tracked from hr.employee and showing twice as I'm posting it as well with a link to the employee